mkeditorpacks/README.md

```diff
@@ -0,0 +1,19 @@
+MkEditorPacks
+=============
+
+This a tool suite to build game packs for map editors.
+
+It is developped for the [Unvanquished](https://unvanquished.net/) project, but must fit any project supported by the editors listed below.
+
+Supported editors
+-----------------
+
+- [GtkRadiant](http://icculus.org/gtkradiant/)
+- [NetRadiant](https://gitlab.com/xonotic/netradiant)
+
+Authors
```

Comment:
Should rather add a proper license file and/or license headers. Ideally we should contact neumond about it as he started working on the scripts. Right now they are effectively copyrighted by him if I'm not mistaken.